src/specific/s_filesystem.c

```diff
@@ -35,3 +37,78 @@ void S_File_CreateDirectory(const char *path)
     mkdir(path, 0664);
 #endif
 }
+
+char *S_File_CasePath(char const *path)
+{
+    assert(path);
+
+    char *current_path = Memory_Alloc(strlen(path) + 2);
+    const char *sep = strchr(path, '/') ? "/" : "\\";
+
+    char *path_copy = Memory_DupStr(path);
+    char *path_piece = path_copy;
+
+    bool relative;
```

Comment:
Looks like this is never actually needed either. It just gets set below based on the type of path, but it's never used beyond that.